src/evm/protocol/vm/state.rs

```diff
@@ -172,9 +209,19 @@ where
         Ok(())
     }
 
-    /// Retrieves the sell amount limit for a given pair of tokens, where the first token is treated
-    /// as the sell token and the second as the buy token. The order of tokens in the input vector
-    /// is significant and determines the direction of the price query.
+    /// Retrieves the sell amount limit for a given pair of tokens and the given overwrites.
```

Comment:
in the end.. what is a sell mount limit?? a limit sell amount that the pool can trade? maybe we can make that more obvious here